Sources/Interaction/Style/InteractorStyleManipulator/index.js

```diff
@@ -239,11 +239,11 @@ function vtkInteractorStyleManipulator(publicAPI, model) {
     if (model.currentRenderer && model.currentManipulator) {
       // When an interaction is active, we should not change the renderer being
       // interacted with.
-    } else {
+    } else if (pos) {
       publicAPI.findPokedRenderer(pos.x, pos.y);
     }
 
```

Comment:
Can you instead do

else {
  publicAPI.findPokedRenderer((pos ? pos.x : 0), (pos ? pos.y : 0));

It is important to have the posed renderer always be set